_docker/drupal/dev/run-drupal.sh

```diff
@@ -25,7 +25,7 @@ if [[ $REGISTRY_REDHAT_IO_SKIP_LOGIN != true ]]; then
   docker login registry.redhat.io -u "${REGISTRY_REDHAT_IO_USERNAME}" -p "${REGISTRY_REDHAT_IO_PASSWORD}"
 fi
 docker pull docker-registry.upshift.redhat.com/developers/drupal-data:latest
-docker pull images.paas.redhat.com/rhdp/developer-base:rhel-76.3
+docker pull images.paas.redhat.com/rhdp/developer-base:rhel-76.4
```

Comment:
I need to understand if this change is alright based upon Rob Blake's recent PR to update the base image version elsewhere in the codebase

Reply:
Rob confirmed that these should be updated to 76.4